server/src/resolvers/mutations/index.ts

```diff
@@ -30,7 +41,7 @@ const Mutation: MutationResolvers = {
     });
 
     extras.set('description', args.description);
-    extras.set('ipfs_hash', args.ipfs_hash);
+    extras.set('ipfs_cid', args.ipfs_cid);
```

Comment:
I believe, we have to make a decision on how to handle the extra fields that are not part of the tzip-12 standard. 

There are two options:
- pass them to the contract as "extras" map 
- upload them to IPFS and pass a CID to the contract

If we go with the second option there is  still a choice:
- upload them on the client side and pass a CID to the server and the contract
- pass "extras" as JSON to the server and let the server upload them to IPFS and pass a CID to the contract

For now, I believe it is fine as it is.